boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -175,6 +185,9 @@ config SPI_DW
 config SPI_0
 	def_bool y
 
+config SPI_DW_FIFO_DEPTH
```

Comment:
This config should not be in the board's config file. Better not to have this this config at all -- if there is a compelling reason to have this, it should rather be in the SoC's Kconfig as this is a configuration of SPI master controller in SoC.

Reply:
ok. i'll move it to soc's Kconfig.

Reply:
Done